charts/microgateway/templates/_helpers.tpl

```diff
@@ -107,6 +107,21 @@ return true if license secret should be mounted
 {{- end -}}
 {{- end -}}
 
+{{/*
```

Comment:
just out of curiosity: why does the filename start with a underscore?

Reply:
That's a Helm convention. Files starting with _ are assumed not to contain a manifest and will not be rendered to a K8s object definitions. The file is created when initializing a new Chart using `helm create my_chart` see https://helm.sh/docs/chart_template_guide/named_templates/ for more information.